### DISCUSSION TITLE
Random temperature changes

### DISCUSSION THREAD

**Discussion by raphthibert** [jmcollin78/versatile_thermostat#575] [Q&A]
Hello, 
On one of my thermostat, the set temperature is changing sometimes : it is in eco mode (via scheduler) and changes to the value of comfort mode (without the comfort icon active). 

I have others thermostat without issue...

Thank you for the help!

**Comment by raphthibert** [comment]
please anyone?

**Comment by jmcollin78** [comment]
Hello @raphthibert ,

Please help us to understand your case. We need at least the VTherm attributes (Development Tools / State), to see how your VTherm is configured.
